pkg/build/builder/docker.go

```diff
@@ -66,7 +68,14 @@ func (d *DockerBuilder) Build() error {
 	if err != nil {
 		return err
 	}
-	sourceInfo, err := fetchSource(d.dockerClient, buildDir, d.build, initialURLCheckTimeout, os.Stdin, d.gitClient)
+	startTime := unversioned.Now()
```

Comment:
@bparees I still think that we should be using something like a helper function and closure, for readability and maintainability, to avoid blow-up of 1 line to 8 at measurement points.